packages/fxa-settings/README.md

```diff
@@ -1,75 +1,182 @@
 # Firefox Accounts Settings
 
+This documentation is up to date as of 2020-06-08.
+
 ## Development
 
-- `npm run start|stop|restart` to start, stop, and restart the server as a PM2 process
-- `npm run build` to create a production build
+- `yarn start|stop|restart` to start, stop, and restart the server as a PM2 process
+- `yarn build` to create a production build
+- `yarn test` to run unit tests
+- `yarn storybook` to open Storybook
 
 ### External imports
 
-You can import React components into this project. This is currently restricted to `fxa-react`:
+You can import React components into this project from other React projects. This is currently restricted to `fxa-react`:
 
 ```javascript
 // e.g. assuming the component HelloWorld exists
 import HelloWorld from 'fxa-react/components/HelloWorld';
 ```
 
+Components from other React packages can be moved into `fxa-react` to be shared across multiple packages.
+
 ### Styling components
 
 #### Tailwind
 
-The package uses [Tailwind CSS](https://tailwindcss.com/) for the bulk of its styles. If you're not familiar you're encouraged to [learn more](https://tailwindcss.com/docs/utility-first), but the idea is simple: use predefined classes on elements to layer on individual styles. **You can accomplish most of your styling needs with the classes provided.**
+At the time of writing, the `fxa-settings`, `fxa-react`, and `fxa-admin-panel` packages are setup to share a [Tailwind CSS](https://tailwindcss.com/) configuration file found in the `fxa-react` package. If you're not familiar with Tailwind, look through [their documentation](https://tailwindcss.com/docs) to get an idea of what [utility-first](https://tailwindcss.com/docs/utility-first) (Atomic CSS) is and what you can expect while using it. The general idea is simple: use single-purpose classes on elements to layer styles until the design is achieved. **You can accomplish almost all of your styling needs with classes provided by Tailwind's default configuration or through adding them in the configuration file.**
```

Comment:
I will take it out, I specified this here because I put the "This documentation is up to date as of 2020-06-08." note at the top and I imagine we're going to want to use Tailwind in payments and likely also the content server after the settings project. Maybe instead of "at the time of writing" I'll mention that we plan to use the same config across all of our FE packages.